Gemfile

```diff
@@ -70,7 +71,6 @@ end
 group :test do
   gem "minitest", require: false
   gem "capybara", "~> 3.35"
-  gem "factory_bot_rails"
```

Comment:
can you please explain why are we moving this?

Reply:
:thinking: When factory bot is in development group as well, you can use it in console during development for local hacking.

Reply:
Yes exactly, I was trying to fix a broken factory and wanted to try it in the development console. And in general I think it's a good idea to be able to quickly create objects for testing, e.g. `FactoryBot.create(:user)`